src/arch/arm/object/vcpu.c

```diff
@@ -430,7 +403,7 @@ exception_t decodeVCPUInjectIRQ(cap_t cap, unsigned int length, word_t *buffer)
         current_syscall_error.type = seL4_RangeError;
         return EXCEPTION_SYSCALL_ERROR;
     }
-    if (priority > 31) {
+    if (priority > GIC_MAX_PRIO) {
         current_syscall_error.type = seL4_RangeError;
         current_syscall_error.rangeErrorMin = 0;
         current_syscall_error.rangeErrorMax = 31;
```

Comment:
I guess `current_syscall_error.rangeErrorMax` should also become `GIC_MAX_PRIO` in that case.

Reply:
agreed.